gensim/models/word2vec.py

```diff
@@ -432,6 +432,8 @@ def __init__(
 
         """
 
+        self.load = call_on_class_only
```

Comment:
If the serialization tests pass, and the exception message is understandable if it's ever called (by mistake), I don't see what other clarity would come from binding the method. That'd just make it look more like an instance-specific operation... which it specifically is not.